app/src/main/java/com/example/kuba10/mypokemonplaces/ListFragment/FirebaseListFragment.java

```diff
@@ -1,151 +0,0 @@
-    private Query orderByChild;
-
-
-    @BindView(R.id.listRecycler)
-    RecyclerView recyclerView;
-
-
-    public static FirebaseListFragment newInstance() {
-        return new FirebaseListFragment();
-    }
-
-    @Override
-    public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        fragmentListener = (FragmentListener) getActivity();
-
-    }
-
-    @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container,
-                             Bundle savedInstanceState) {
-        View view = inflater.inflate(R.layout.fragment_list, container, false);
-        ButterKnife.bind(this, view);
-
-
-        prepareQuery();
-        setUpFirebaseAdapter();
-
-        return view;
-    }
-
-    @Override
-    public void onStart() {
-        super.onStart();
-    }
-
-    private void setUpFirebaseAdapter() {
-
-        mFirebaseAdapter = new AdapterForTouchAndFirebase(PokePlace.class,
-                R.layout.place_card_layout, CardViewHolder.class,
-                orderByChild, this, this);
-
-        recyclerView.setHasFixedSize(true);
-        LinearLayoutManager layoutManager = new LinearLayoutManager(getActivity());
-        recyclerView.setLayoutManager(layoutManager);
-        recyclerView.setAdapter(mFirebaseAdapter);
-
-        ItemTouchHelper.Callback callback = new SimpleItemTouchHelperCallback(mFirebaseAdapter);
-        mItemTouchHelper = new ItemTouchHelper(callback);
-        mItemTouchHelper.attachToRecyclerView(recyclerView);
-    }
-
-    private void prepareQuery() {
-        orderByChild = FirebaseDatabase.getInstance()
-                .getReference()
-                .child(Constants.PLACES)
-                .orderByChild(Constants.FIREBASE_QUERY_INDEX);
-    }
-
-    @Override
-    public void onStartDrag(RecyclerView.ViewHolder viewHolder) {
-        mItemTouchHelper.startDrag(viewHolder);
-    }
-
-    @Override
-    public void onPause() {
-
-        super.onPause();
-        mFirebaseAdapter.cleanup();
-
-    }
-
-    public void dismiss() {
-
-        getActivity().getSupportFragmentManager()
-                .beginTransaction()
-                .setCustomAnimations(R.anim.enter, R.anim.exit, R.anim.pop_enter, R.anim.pop_exit)
-                .remove(this).commit();
-    }
-
-    public void findPosition(PokePlace place) {
-
-        fragmentListener.findPlaceFromList(place);
-
-
-    }
-
-
-    @Override
-    public void onDetach() {
-        super.onDetach();
-        fragmentListener = null;
-        orderByChild = null;
-        mItemTouchHelper = null;
-        mFirebaseAdapter = null;
-
-    }
-
-    public void refreshList(int size) {
-    }
-
-    public void openDetails(Fragment fragment) {
-        fragmentListener.openFragment(fragment);
-    }
-
-    public Query getQuery() {
-        return orderByChild;
-    }
-
-    public ArrayList<PokemonGo> sendPokemonListToAdapter() {
-        return fragmentListener.getPokemonList();
```

Comment:
"wyślij listę pokemonów do adaptera" -> "weź listę pokemonów" - czytając kod zupełnie nie wiem co się dzieje :disappointed:

Reply:
Nie chciałem zapisywać listy jako pola - bo nie ejst potrzebna, ale muszę przekazać ją dalej - a ona żyje tylko w klasie głównej do której ma dostęp interface który posiada fragment... więc mogę tak, albo stworzyć obiekt "retrofitConnection" i wziąć z niego get.

Szczerze cała ta historia z listą w 5 miejscach to jakieś nieporozumienie - albo ćwiczeniowo zrobię to tu daggerem jak znajdę chwile, albo poprostu następnym razem będe trzymał się daggera do takich rzeczy